static/app.scss

```diff
@@ -1,14 +1,115 @@
-@import "ams";
-@import "ams-bootstrap";
-@import "ams-ois";
+//
+//  Application specific styling
+//  specific
+//
 
-.borderless tbody td, .borderless thead th {
-  border: 0;
+@import "~stijl/src/styles/ams-stijl";
+
```

Comment:
I suggest commenting the styles a bit, especially styles like breakout zone

Reply:
done